src/components/scenes/Loans/LoanDashboardScene.js

```diff
@@ -112,34 +111,34 @@ export const LoanDashboardScene = (props: Props) => {
     }
   })
 
-  const renderLoanCard = useHandler((item: FlatListItem<string>) => {
-    const loanAccount: LoanAccount = loanAccounts[item.item]
+  const renderLoanCard = useHandler((item: FlatListItem<LoanAccount>) => {
+    const loanAccount: LoanAccount = item.item
 
     const handleLoanPress = () => {
-      navigation.navigate('loanDetails', { loanAccount })
+      navigation.navigate('loanDetails', { loanAccountId: loanAccount.id })
     }
     return (
       <LoanSummaryCard onPress={handleLoanPress} borrowEngine={loanAccount.borrowEngine} iconUri={iconUri} exchangeRates={exchangeRates} key={loanAccount.id} />
     )
   })
 
+  const isCompatibleWalletsAvailable =
```

Comment:
This was meant for 275ab117a7b1c3e6cef334a37d6c4d77dc7af8ea

Reply:
I'll save commenting on any other fixup misplacements because I'm sure this will all be resolved in the rebase.